stdlib/public/SwiftShims/SwiftStddef.h

```diff
@@ -13,7 +13,12 @@
 #ifndef SWIFT_STDLIB_SHIMS_SWIFT_STDDEF_H
 #define SWIFT_STDLIB_SHIMS_SWIFT_STDDEF_H
 
+#if !defined(__clang__)
+#include <stdint.h>
+typedef size_t __swift_size_t;
```

Comment:
I'm not sure that's the point of these headers (or perhaps not the main one). For example, there's a comment in `SwiftStdint.h` that says:

>  // stdint.h is provided by Clang, but it dispatches to libc's stdint.h.  As a
>  // result, using stdint.h here would pull in Darwin module (which includes
>  // libc).  This creates a dependency cycle, so we can't use stdint.h in

I don't think we have this problem with MSVC (confirmed) and perhaps even Clang on Win32(unconfirmed)